src/readers/morphologyASC.cpp

```diff
@@ -72,9 +72,19 @@ class NeurolucidaParser
     NeurolucidaParser& operator=(NeurolucidaParser const&) = delete;
 
     morphio::mut::Morphology& parse() {
+#ifdef _WIN32
         std::ifstream ifs(uri_);
         std::string input((std::istreambuf_iterator<char>(ifs)),
                           (std::istreambuf_iterator<char>()));
+#else
+        std::ifstream ifs(uri_, std::ios::ate);
+        const auto size = ifs.tellg();
+        ifs.seekg(0);
+        std::string input;
+        input.resize(static_cast<size_t>(size));
```

Comment:
you can shave some more time off w/ `input.reserve` b/c then it doesn't need to fill the array, FYI.

Reply:
The problem with reserve is that it does not modify the string size